website/src/content/home/features/features.ts

```diff
@@ -0,0 +1,3 @@
+export const title = "FEATURES";
+export const subtitle =
+  "Build Reliable and Scalable systems with Smart Resources.";
```

Comment:
This is way too long, especially in mobile. I wonder if we should trash it?

Reply:
It doesn't add much value - just have the `FEATURES` small piece of text and then dive straight into the sections.